examples/arithmetics/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "langium-arithmetics-dsl",
   "displayName": "Arithmetics DSL",
-  "version": "3.5.0",
+  "version": "3.6.0",
```

Comment:
Please don't update the package versions - we will do this ourselves when we perform the release.